pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -672,7 +685,7 @@ func (m *kubeGenericRuntimeManager) killContainersWithSyncResult(pod *v1.Pod, ru
 			defer wg.Done()
 
 			killContainerResult := kubecontainer.NewSyncResult(kubecontainer.KillContainer, container.Name)
-			if err := m.killContainer(pod, container.ID, container.Name, "", gracePeriodOverride); err != nil {
+			if err := m.killContainer(pod, container.ID, container.Name, "", "", gracePeriodOverride); err != nil {
```

Comment:
Do we want to use `reasonUnknown` here instead of `""`

Reply:
I was on the fence because it wasn't obvious if it was actually unknown but yeah I can update